<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1010</width>
    <height>678</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PARAVIS Light!</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QSplitter" name="splitter_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QSplitter" name="splitter">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QFrame" name="pipelineFrame">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayoutPipe">
         <item>
          <widget class="QWidget" name="widget" native="true"/>
         </item>
        </layout>
       </widget>
       <widget class="QFrame" name="propFrame">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayoutProp">
         <item>
          <widget class="QWidget" name="widget_2" native="true"/>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>-1</number>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Chosen file:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="currentFileLabel">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="openButton">
        <property name="text">
         <string>Open ...</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <spacer name="horizontalSpacer_6">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="addTabButton">
        <property name="text">
         <string>Add tab ...</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_7">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="detTabButton">
        <property name="text">
         <string>Delete current tab</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_8">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pluginsButton">
        <property name="text">
         <string>Manage Plugins ...</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="sliceButton">
        <property name="text">
         <string>Slice</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="shrinkButton">
        <property name="text">
         <string>Shrink</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="quitButton">
        <property name="text">
         <string> Quit </string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1010</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_file"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuSimple_actions">
    <property name="title">
     <string>Simple actions</string>
    </property>
    <addaction name="actionSlice"/>
    <addaction name="actionShrink"/>
   </widget>
   <widget class="QMenu" name="menuTabs">
    <property name="title">
     <string>Tabs</string>
    </property>
    <addaction name="actionAdd_tab"/>
    <addaction name="actionDelete_tab"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionAuto_apply"/>
    <addaction name="actionShow_prop_widget"/>
    <addaction name="actionShow_pipeline"/>
    <addaction name="actionBuild_filter_menu"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuSimple_actions"/>
   <addaction name="menuOptions"/>
   <addaction name="menuTabs"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen_file">
   <property name="text">
    <string>Open file ...</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionSlice">
   <property name="text">
    <string>Slice</string>
   </property>
  </action>
  <action name="actionShrink">
   <property name="text">
    <string>Shrink</string>
   </property>
  </action>
  <action name="actionAdd_tab">
   <property name="text">
    <string>Add tab</string>
   </property>
  </action>
  <action name="actionDelete_tab">
   <property name="text">
    <string>Delete tab</string>
   </property>
  </action>
  <action name="actionAuto_apply">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Auto apply</string>
   </property>
  </action>
  <action name="actionShow_prop_widget">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Show prop widget</string>
   </property>
  </action>
  <action name="actionShow_pipeline">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Show pipeline</string>
   </property>
  </action>
  <action name="actionBuild_filter_menu">
   <property name="text">
    <string>Build filter menu</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>quitButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>871</x>
     <y>588</y>
    </hint>
    <hint type="destinationlabel">
     <x>439</x>
     <y>210</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>openButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>onFileOpen()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1004</x>
     <y>271</y>
    </hint>
    <hint type="destinationlabel">
     <x>456</x>
     <y>258</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>MainWindow</sender>
   <signal>changedCurrentFile(QString)</signal>
   <receiver>currentFileLabel</receiver>
   <slot>setText(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>456</x>
     <y>258</y>
    </hint>
    <hint type="destinationlabel">
     <x>211</x>
     <y>259</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>shrinkButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>shrink()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>674</x>
     <y>588</y>
    </hint>
    <hint type="destinationlabel">
     <x>456</x>
     <y>258</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliceButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>slice()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>472</x>
     <y>588</y>
    </hint>
    <hint type="destinationlabel">
     <x>504</x>
     <y>338</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pluginsButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>managePlugins()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>279</x>
     <y>588</y>
    </hint>
    <hint type="destinationlabel">
     <x>504</x>
     <y>338</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSlice</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>slice()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>504</x>
     <y>338</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionShrink</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>shrink()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>504</x>
     <y>338</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionOpen_file</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>onFileOpen()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>504</x>
     <y>338</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>504</x>
     <y>338</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAdd_tab</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>addTab()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>504</x>
     <y>338</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionDelete_tab</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>deleteTab()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>504</x>
     <y>338</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addTabButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>addTab()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>350</x>
     <y>430</y>
    </hint>
    <hint type="destinationlabel">
     <x>504</x>
     <y>338</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>detTabButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>deleteTab()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>750</x>
     <y>430</y>
    </hint>
    <hint type="destinationlabel">
     <x>504</x>
     <y>338</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAuto_apply</sender>
   <signal>toggled(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>autoApplyCheck(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>504</x>
     <y>338</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionShow_pipeline</sender>
   <signal>toggled(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>showPipeline(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>504</x>
     <y>338</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionShow_prop_widget</sender>
   <signal>toggled(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>showProp(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>504</x>
     <y>338</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tabWidget</sender>
   <signal>currentChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>currentTabChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>894</x>
     <y>101</y>
    </hint>
    <hint type="destinationlabel">
     <x>504</x>
     <y>338</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionBuild_filter_menu</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>onBuildFilterMenu()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>504</x>
     <y>338</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <signal>changedCurrentFile(QString)</signal>
  <slot>onFileOpen()</slot>
  <slot>slice()</slot>
  <slot>shrink()</slot>
  <slot>managePlugins()</slot>
  <slot>addTab()</slot>
  <slot>deleteTab()</slot>
  <slot>autoApplyCheck(bool)</slot>
  <slot>showPipeline(bool)</slot>
  <slot>showProp(bool)</slot>
  <slot>currentTabChanged(int)</slot>
  <slot>onBuildFilterMenu()</slot>
 </slots>
</ui>
